neo/rawio/axonrawio.py

```diff
@@ -489,8 +501,8 @@ def parse_axon_soup(filename):
                         ADCInfo[key] = val[0]
                     else:
                         ADCInfo[key] = np.array(val)
-                ADCInfo["ADCChNames"] = strings[ADCInfo["lADCChannelNameIndex"] - 1]
-                ADCInfo["ADCChUnits"] = strings[ADCInfo["lADCUnitsIndex"] - 1]
+                ADCInfo["ADCChNames"] = strings[ADCInfo["lADCChannelNameIndex"]]
+                ADCInfo["ADCChUnits"] = strings[ADCInfo["lADCUnitsIndex"]]
```

Comment:
Why do we remove this *-1* ?
I am not sure to follow the logic why the new string parsing change the one or zero based indexing.

Reply:
Hey Sam. It's because the string actually has an id at the beginning `[uModifierNameIndex, uCreatorNameIndex, uProtocolPathIndex, lFileComment, lADCCChannelNames, lADCUnitsIndex
lDACChannelNameIndex, lDACUnitIndex, lDACFilePath, nLeakSubtractADC]`

Our previous parsing started at `uCreatorNameIndex` which meant we had to subtract 1 to get the correct indexing. The new indexing starts at the appropriate index `uModifernameIndex` which means that don't need to do the shifting.